subsys/caf/modules/power_manager.c

```diff
@@ -143,6 +144,11 @@ static void system_off_on_error(void)
 
 static void power_down(struct k_work *work)
 {
+	if (keep_alive_flag) {
```

Comment:
is it still valid? I don't get what's wrong

Reply:
It's not valid already. When Marek wrote it the check was done only after expiration of power down timeout so in the worst case scenario power down timeout could be doubled. Now it can only be increased by 1 second.